src/lib/dom.generated.d.ts

```diff
@@ -14278,7 +14313,7 @@ declare var location: Location;
 declare var locationbar: BarProp;
 declare var menubar: BarProp;
 declare var msCredentials: MSCredentials;
-declare var name: string;
+declare const name: never;

```

Comment:
What is this all about?

Reply:
See https://github.com/Microsoft/TypeScript/issues/9850